package.json

```diff
@@ -31,54 +31,51 @@
     "@storybook/addon-links": "^3.2.10",
     "@storybook/addon-options": "3.2.10",
     "@storybook/react": "^3.2.11",
-    "babel-cli": "^6.26.0",
-    "babel-core": "6.7.2",
-    "babel-loader": "^7.1.2",
+    "babel-cli": "^6.0.0",
+    "babel-core": "^6.0.0",
+    "babel-jest": "^23.6.0",
+    "babel-loader": "^7.0.0",
     "babel-plugin-module-resolver": "2.7.1",
     "babel-plugin-transform-define": "^1.3.0",
     "babel-plugin-transform-es2015-destructuring": "6.9.0",
     "babel-plugin-transform-object-rest-spread": "6.8.0",
     "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-env": "^1.7.0",
     "babel-preset-es2015": "6.6.0",
     "babel-preset-react": "6.5.0",
     "babel-runtime": "^6.26.0",
     "emoji-datasource": "4.0.4",
     "emojilib": "^2.2.1",
     "inflection": "1.10.0",
-    "jasmine-core": "^2.5.2",
-    "karma": "^1.4.0",
-    "karma-chrome-launcher": "^2.0.0",
-    "karma-cli": "^1.0.1",
-    "karma-jasmine": "^1.1.0",
-    "karma-webpack": "^2.0.4",
+    "jest": "^23.0.0",
     "mkdirp": "0.5.1",
     "prettier": "1.11.1",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
+    "react-test-renderer": "^16.8.4",
     "rimraf": "2.5.2",
     "size-limit": "^0.11.4",
     "webpack": "^3.6.0"
   },
   "scripts": {
-    "clean": "rm -rf dist/ dist-es/",
+    "clean": "rm -rf dist/ dist-es/ dist-modern/",
     "build:data": "node scripts/build-data",
     "build:dist": "npm run build:cjs && npm run build:es && npm run build:modern",
-    "build:cjs": "BABEL_ENV=legacy-cjs babel src --out-dir dist --copy-files",
-    "build:es": "BABEL_ENV=legacy-es babel src --out-dir dist-es --copy-files",
-    "build:modern": "babel src --out-dir dist-modern --copy-files",
+    "build:cjs": "BABEL_ENV=legacy-cjs babel src --out-dir dist --copy-files --ignore '**/*.test.js'",
+    "build:es": "BABEL_ENV=legacy-es babel src --out-dir dist-es --copy-files --ignore '**/*.test.js'",
+    "build:modern": "babel src --out-dir dist-modern --copy-files --ignore '**/*.test.js'",
     "build:docs": "cp css/emoji-mart.css docs && webpack --config ./docs/webpack.config.js",
     "build": "npm run clean && npm run build:dist",
-    "watch": "BABEL_ENV=cjs babel src --watch --out-dir dist --copy-files",
+    "watch": "BABEL_ENV=cjs babel src --watch --out-dir dist --copy-files --ignore '**/*.test.js'",
     "start": "npm run watch",
-    "stats": "webpack --config ./spec/webpack.config.js --json > spec/stats.json",
```

Comment:
I could not find a way to keep this `stats` script without maintaining a webpack config merely to generate it... seems not worth it.